translation by zhutianyu (#363)
Oneflow-Inc/oneflow-documentation: en/docs/basics/04_build_network.md

# Build Neural Network

> The layers of A neural network can be constructed using API in namespace [oneflow.nn](https://oneflow.readthedocs.io/en/master/nn.html), which provide common modules (e.g.,[oneflow.nn.Conv2d](https://oneflow.readthedocs.io/en/master/nn.html?highlight=oneflow.nn.Conv2D#oneflow.nn.Conv2d), [oneflow.nn.ReLU](https://oneflow.readthedocs.io/en/master/nn.html?highlight=oneflow.nn.ReLU#oneflow.nn.ReLU), etc.) needed to build neural network. All Module classes used to build networks inherit from [oneflow.nn.Module](https://oneflow.readthedocs.io/en/master/module.html#oneflow.nn.Module), and multiple simple Module can be combined to form a more complex Module. In this way, users can easily build and manage complex neural networks.

Reviewer: "The layers of A neural network " -> The layers of a neural network 
“All Module classes” -> All moudle classes

注意大小写，其它好像还有很多类似的问题，自己全文检查和修正。不再重复

Reviewer: 提取下第一句的主干会发现是这样的：
> The layers of A neural network can be constructed using API in namespace oneflow.nn,  which provide common modules needed to build nerual network.

后面的  "needed to build nerual network" 其实就多余了，所以整个句子就怪怪的。

```python
import oneflow as flow
import oneflow.nn as nn
```

> ## Define Module Class

Reviewer: ```suggestion
## Define the Class
```
很多不知道怎么翻的，可以先认真研究下 PyTorch 的这篇https://pytorch.org/tutorials/beginner/basics/buildmodel_tutorial.html
毕竟我们中文版参考了不少 PyTorch

> Common Module Class are provided in namespace `oneflow.nn` , and we can use them directly or build neural networks based on them by custom Module classes. The build process includes the following:

Reviewer: PyTorch 的 tutorial 中有类似的句子：The torch.nn namespace provides ……

现有的翻译过于追求直译，问题比较多

> - Build a Class that inherits from `oneflow.nn.Module` 

Reviewer: ```suggestion
- Define a Class that inherits from `oneflow.nn.Module` 
```

> - Implement the `__init__` method of Class and construct the structure of neural network

Reviewer: ```suggestion
- Initialize the neural network layers in `__init__` method.
```

> - Implement the class's 'forward' method, which evaluates the input to Module

Reviewer: ```suggestion
- Implement the `forward` method, which takes the input and then executes the computation
```

markdown 里的代码块，是用反引号，不是单引号
